documentation/modules/configuring/proc-config-mirrormaker2-replication.adoc

```diff
@@ -132,6 +132,8 @@ spec:
         enabled: true
       config:
         heartbeats.topic.replication.factor: 1 # <29>
+        replication.policy.separator: "."
```

Comment:
`separator` is only a configuration for `DefaultReplicationPolicy`, it's not used by `IdentityReplicationPolicy`

Reply:
I've removed the separator property and extended the callout description for the policy class.

> <26> Adds a policy that overrides the automatic renaming of remote topics. Instead of prepending the name with the name of the source cluster, the topic retains its original name. This optional setting is useful for active/passive backups and data migration. The property must be specified for all connectors. For bidirectional (active/active) replication, use the `DefaultReplicationPolicy` class to automatically rename remote topics and specify the `replication.policy.separator` property for all connectors to add a custom separator.